backendCore/dl/dl_model_parser.py

```diff
@@ -0,0 +1,84 @@
+from sklearn.tree import *
+from sklearn.ensemble import *
+from sklearn.linear_model import *
+from sklearn.svm import *
+from sklearn.naive_bayes import *
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+import torchvision.transforms as transforms
+
+
+def parse_deep_user_architecture(user_model):
+    """
+    new:
+    Each layer in the user_model has this format:
+    {"value": "LAYER_NAME", "parameters": [PARAMETER_1, PARAMETER_2, ...]}
+
+    Eg: [{"value": "LINEAR", "parameters": [50, 10]}] should become [nn.Linear(in_features=50, out_features=10)]
```

Comment:
does the user know about in_features = X, out_features = Y on our frontend? On our backend, do we ensure that the list of [PARAMETER_1, PARAMETER_2, etc] stays ordered so that we map the parameter to the right thing? @dwu359